src/main/java/by/custom/utilcalculator/service/UserProgressManager.java

```diff
@@ -33,93 +35,123 @@ public void createNewUserProgress(final String chatID) throws UtilsborException
     }
 
     public String processCarOrigin(final String requestingCommand, final String chatID) throws UtilsborException {
-        UserProgress userProgress;
+        final UserProgress userProgress;
         userProgress = userProgressStorage.get(chatID);
-        String message;
-        switch (requestingCommand) {
-            case Command.EAES -> userProgress.setCountryOrigin(CountryOrigin.EAES);
-            case Command.OTHER_COUNTRIES -> userProgress.setCountryOrigin(CountryOrigin.OTHER);
-
+        final String message;
+
+        if (Objects.equals(requestingCommand, Command.EAES.getCommand())) {
```

Comment:
It's betteruse type `Command` for the argument `requestingCommand`  instead of casting `Command` to `String`.

Reply:
Fixed it according to your advice, thank you